src/socket-activate/socket-activate.c

```diff
@@ -298,9 +298,9 @@ static int help(void) {
         if (r < 0)
                 return log_oom();
 
-        printf("%s [OPTIONS...]\n"
-               "\n%sListen on sockets and launch child on connection.%s\n"
-               "\nOptions:\n"
+        printf("%1$s [OPTIONS...]\n"
+               "\n%2$sListen on sockets and launch child on connection.\n"
```

Comment:
please reset the ansi color to default before the line break. i.e. ansi_normal() needs to be output right before the `\n`